povm_toolbox/quantum_info/product_dual.py

```diff
@@ -73,38 +73,18 @@ def is_dual_to(self, frame: BaseFrame) -> bool:
         #      have not implemented the check for this. Then we should raise an NotImplementedError.
         raise NotImplementedError
 
-    def optimize(self, frame: BaseFrame, **options) -> None:
-        """Optimize the dual to `frame` inplace.
-
-        Args:
-            frame: The primal frame to which ``self`` is a dual.
-            options: keyword arguments specifying how to optimize ``self``.
-        """
-        state = options.get("state", None)
-        if not isinstance(frame, ProductFrame):
-            raise NotImplementedError
-        if state is not None:
-            axes = np.arange(len(frame.sub_systems), dtype=int)
-            if isinstance(state, (Statevector, DensityMatrix)):
-                state = Operator(state)
-            elif not isinstance(state, Operator):
-                raise TypeError
-            joint_prob: np.ndarray = frame.analysis(state)  # type: ignore
-            for i, sub_system in enumerate(frame.sub_systems):
-                marg_prob = joint_prob.sum(axis=tuple(np.delete(axes, [i])))
-                if not all(marg_prob):
-                    marg_prob += 1e-3
-                self[sub_system].optimize(frame=frame[sub_system], alphas=tuple(marg_prob))
-
     @classmethod
     def build_dual_from_frame(
         cls, frame: BaseFrame, alphas: tuple[tuple[float, ...] | None, ...] | None = None
     ) -> ProductDUAL:
         """Construct a dual frame to another frame.
 
         Args:
-            frame: The primal frame from which we will build the dual frame.
-            alphas: TODO.
+            frame: the primal frame from which we will build the dual frame.
+            alphas: parameters of the local frame super-operators used to build
+                the local dual frames which form together the product dual frame.
+                If None, the parameters are set as the traces of each local operator
+                in each of the primal frames.
```

Comment:
This should be aligned with the docstring o the base class (as much as possible).